### PR TITLE
Add convert names case to make federated schema

### PR DESCRIPTION
**Pull request by a-rhote**
Attempt 2, with more imports this time :D

### CODE REVIEW

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Okay, now those are running

File: ariadne/contrib/federation/schema.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
     GraphQLUnionType,
 )
 
+from ... import SchemaNameConverter
```
Comment (rafalp) [inline ariadne/contrib/federation/schema.py:13]: Make this import use `schema_names` module and reorder imports.

**Review by rafalp — CHANGES_REQUESTED** [review]

File: ariadne/contrib/federation/schema.py (hunk 1 above)
Comment (a-rhote) [reply ariadne/contrib/federation/schema.py:13]: Can do

Comment (a-rhote) [reply ariadne/contrib/federation/schema.py]: Should be updated now

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Thank you!

**Pull request merged**